contracts/Joint.sol

```diff
@@ -182,54 +187,190 @@ contract Joint {
         return string(abi.encodePacked("JointOf", ab));
     }
 
-    function harvest() external onlyGuardians {
+    function estimatedTotalAssetsInToken(address token)
```

Comment:
who's going to call this?

Reply:
The providers can use this in estimatedTotalAssets